chaco/tools/simple_zoom.py

```diff
@@ -68,27 +78,48 @@ class SimpleZoom(AbstractOverlay, ToolHistoryMixin, BaseZoomTool):
     #: Conversion ratio from wheel steps to zoom factors.
     wheel_zoom_step = Float(1.0)
 
+<<<<<<< HEAD
```

Comment:
another merge conflict

Reply:
looks like there are a few more in this module

Reply:
Oh yikes! 😓 thank you for catching these!